src/api/users.coffee

```diff
@@ -21,13 +21,13 @@ exports.authenticate = (email) ->
     user = yield User.findOne(email: email).exec()
 
     if not user
-      utils.logAndSetResponse this, 'not found', "Could not find user by email #{email}", 'info'
+      util.logAndSetResponse this, 'not found', "Could not find user by email #{email}", 'info'
```

Comment:
hmmm, I think this will break master if we merge this branch. This override the fixed version there. You may need to pull from master and make sure this reflects the fixed changes.